Mathlib/Tactic/NormNum/Core.lean

```diff
@@ -121,7 +108,7 @@ structure IsRat [Ring α] (a : α) (num : ℤ) (denom : ℕ) where
   /-- The denominator is invertible. -/
   inv : Invertible denom
   /-- The element is equal to the fraction with the specified numerator and denominator. -/
-  eq : a = num * ⅟denom
+  eq : a = num * ⅟ denom
```

Comment:
We may want to define this syntax with `prefix` rather than `notation` here: https://github.com/leanprover-community/mathlib4/blob/089d3a2d75176d354a306bb7f81bf3b0f79ca5c6/Mathlib/Algebra/Invertible.lean#L79-L81

As is, we need to insert this space.

Reply:
That seems reasonable to me.  Are there any other side effects of making something `prefix` rather than `notation`?

Reply:
The usages of the syntax in that file (e.g. here: https://github.com/leanprover-community/mathlib4/blob/089d3a2d75176d354a306bb7f81bf3b0f79ca5c6/Mathlib/Algebra/Invertible.lean#L94) would then need to *remove* their spaces. So it's not a one-line change. Unless someone knows a way to define the syntax that accepts both the space and no-space version?

Reply:
@ChrisHughes24 was there a reason for using `notation` rather than `prefix` here? It seems like `prefix:max` would be the most analogous to the `Inv.inv` notation: https://github.com/leanprover-community/mathlib4/blob/cac405f4bb86f998a4d71cb6836b2a05afb4bcea/Mathlib/Algebra/Group/Defs.lean#L111

Reply:
I was mistaken-- the `prefix:max` version does not actually seem to require other changes, so I went ahead and did that.